mailpoet/assets/js/src/automation/editor/components/inserter-popover/index.tsx

```diff
@@ -1,28 +1,66 @@
+import { useCallback, useRef, useState } from 'react';
 import { Popover } from '@wordpress/components';
-import { dispatch, useSelect } from '@wordpress/data';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { Hooks } from 'wp-js-hooks';
+import { PremiumModal } from 'common/premium_modal';
 import { Inserter } from '../inserter';
-import { store } from '../../store';
+import { Item } from '../inserter/item';
+import { storeName } from '../../store';
 
 export function InserterPopover(): JSX.Element | null {
```

Comment:
`JSX.Element | null`

out of curiosity: Isn't `null` a valid `JSX.Element`?

Reply:
It's [a bit compicated](https://stackoverflow.com/a/58123882/1236795), but no, `null` is not included in `JSX.Element`, it's only included in `React.ReactNode` but JSX element [seems to be the best consensus](https://github.com/typescript-cheatsheets/react/issues/129) for typing component return types.